setup.py

```diff
@@ -27,11 +27,16 @@
       keywords='irods',
       classifiers=[
                    'License :: OSI Approved :: BSD License',
-                   'Development Status :: 4 - Beta',
+                   'Development Status :: 5 - Production/Stable',
                    'Programming Language :: Python :: 2.7',
                    'Programming Language :: Python :: 3.4',
                    'Programming Language :: Python :: 3.5',
                    'Programming Language :: Python :: 3.6',
+                   'Programming Language :: Python :: 3.7',
+                   'Programming Language :: Python :: 3.8',
+                   'Programming Language :: Python :: 3.9',
+                   'Programming Language :: Python :: 3.10',
+                   'Programming Language :: Python :: 3.11',
```

Comment:
Have we tried this on 3.12?

Reply:
i wrote the new password booth on 3.12.  so, yes!

Reply:
added